tests/sympc/tensor/replicatedshare_tensor_test.py

```diff
@@ -137,3 +139,17 @@ def test_hook_property(get_clients) -> None:
 
     assert (rst.T.shares[0] == x.T).all()
     assert (rst.T.shares[1] == y.T).all()
+
+
+def test_rst_distribute_reconstruct(get_clients) -> None:
```

Comment:
Q: Can we be more specific with the test like checking the number of shares each party is holding?

Reply:
Sure, that's a good suggestion. Will do that.

Reply:
@aanurraj Done.